tests/common/DotNet.cs

```diff
@@ -113,7 +113,7 @@ public static ExecutionResult AssertNew (string outputDirectory, string template
 			return new ExecutionResult (output, output, rv.ExitCode);
 		}
 
-		public static ExecutionResult Execute (string verb, string project, Dictionary<string, string>? properties, bool assert_success = true, string? target = null, bool? msbuildParallelism = null)
+		public static ExecutionResult Execute (string verb, string project, Dictionary<string, string>? properties, bool assert_success = true, string? target = null, bool? msbuildParallelism = null, TimeSpan? timeout = null, bool? quiet = null)
```

Comment:
Why a bool? and not default to false? I hate having to deal with nullability and bools.

Reply:
Fixed.